# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/DecoderFactory.cpp
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd image/DecoderFactory.cpp
--- a/image/DecoderFactory.cpp	Fri Jan 27 20:35:34 2023 +0000
+++ b/image/DecoderFactory.cpp	Fri Jan 27 22:54:08 2023 +0200
@@ -217,7 +217,7 @@
   }
 
   MOZ_ASSERT(aType == DecoderType::GIF || aType == DecoderType::PNG ||
-                 aType == DecoderType::WEBP || aType == DecoderType::AVIF,
+                 aType == DecoderType::WEBP,
              "Calling CreateAnimationDecoder for non-animating DecoderType");
 
   // Create an anonymous decoder. Interaction with the SurfaceCache and the
@@ -272,7 +272,7 @@
   // rediscover it is animated).
   DecoderType type = aDecoder->GetType();
   MOZ_ASSERT(type == DecoderType::GIF || type == DecoderType::PNG ||
-                 type == DecoderType::WEBP || type == DecoderType::AVIF,
+                 type == DecoderType::WEBP,
              "Calling CloneAnimationDecoder for non-animating DecoderType");
 
   RefPtr<Decoder> decoder = GetDecoder(type, nullptr, /* aIsRedecode = */ true);